### PR TITLE
YouTube Regrets 2022 - Localization fixes

### PR DESCRIPTION
**Pull request by TheoChevalier**
- Localizing video titles in the timeline
- Further l10n fixes in other area of the page

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html [hunk 1]
```diff
@@ -87,20 +87,27 @@ <h2>{% trans 'A week later' %}</h2>
...
+        {% trans 'A Student Felt A Sharp Pain In Her Side. This Is How Her Organs Shut Down.' as localized_title_4_3 %}
+        {% trans 'A Mom Drank 3 Gallons Water In 2 Hours. This is What Happened to Her Brain.' as localized_title_4_1 %}
+        {% trans 'A Gamer Drank 12 Energy Drinks In 10 Minutes. This Is What Happened To His Organs.' as localized_title_4_4 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-1" size="small" image='scene4/2.jpg' speed="3" text=localized_title_4_2 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4 tw-row-start-2" size="small" image='scene4/3.jpg' speed="2" text=localized_title_4_3 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-3" size="small" image='scene4/1.jpg' speed="2" text=localized_title_4_1 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-4" image='scene4/4.jpg' speed="3" text=localized_title_4_4 %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html:97]: **Question** Is there a particular reason we are not having the variable definition and it's use closer together? (trans, include, trans, include, ...)

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html [hunk 2]
```diff
@@ -87,20 +87,27 @@ <h2>{% trans 'A week later' %}</h2>
...
-        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-3" image='scene4/7.jpg' speed="6" text="A Toddler Drank His Mom's Essential Oils. This Is What Happened To His Brain." %}
+        {% trans 'A Woman Drank "35% Food Grade Hydrogen Peroxide." This Is What Happened To Her Brain.' as localized_title_4_5 %}
+        {% trans 'A Student Drank 2 Bottles Over The Counter Cough Medicine. This Is What Happened To Her Brain.' as localized_title_4_6 %}
+        {% trans "A Toddler Drank His Mom's Essential Oils. This Is What Happened To His Brain." as localized_title_4_7 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 tw-row-start-1" image='scene4/5.jpg' speed="4" text=localized_title_4_5 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4 tw-row-start-2" size="small" image='scene4/6.jpg' speed="5" text=localized_title_4_6 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-3" image='scene4/7.jpg' speed="6" text=localized_title_4_7 %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html:106]: See above

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html [hunk 3]
```diff
@@ -114,29 +121,43 @@ <h2>{% trans 'A month later' %}</h2>
...
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-3" size="small" image='scene5/2.jpg' speed="5" text=localized_title_5_2 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-4" image='scene5/19.jpg' speed="4" text=localized_title_5_19 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-5" size="small" image='scene5/5.jpg' speed="3" text=localized_title_5_5 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  tw-row-start-6" size="small" image='scene5/6.jpg' speed="2" text=localized_title_5_6 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-7" size="small" image='scene5/4.jpg' speed="4" text=localized_title_5_4 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 tw-row-start-8" image='scene5/16.jpg' speed="3" text=localized_title_5_16 %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html:139]: See above

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html [hunk 4]
```diff
@@ -114,29 +121,43 @@ <h2>{% trans 'A month later' %}</h2>
...
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  medium:tw-col-start-5 tw-row-start-3" image='scene5/8.jpg' speed="3" text=localized_title_5_8 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 tw-row-start-4" image='scene5/7.jpg' speed="2" text=localized_title_5_7 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  medium:tw-col-start-5 tw-row-start-5" image='scene5/18.jpg' speed="5" text=localized_title_5_18 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  medium:tw-col-start-5 tw-row-start-6" size="small" image='scene5/3.jpg' speed="4" text=localized_title_5_3 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 tw-row-start-7" image='scene5/20.jpg' speed="3" text=localized_title_5_20 %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html:155]: See above

File: network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html [hunk 5]
```diff
@@ -150,33 +171,53 @@ <h2>{% trans 'Two months later' %}</h2>
...
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  tw-row-start-8" size="small" image='scene6/5.jpg' speed="4" text=localized_title_6_5 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-8" size="small" image='scene6/6.jpg' speed="2" text=localized_title_6_6 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 tw-row-start-9" image='scene6/15.jpg' speed="4" text=localized_title_6_15 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-10" size="small" image='scene6/9.jpg' speed="2" text=localized_title_6_9 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-2 medium:tw-col-start-3 medium:tw-col-start-4  tw-row-start-11" size="small" image='scene6/11.jpg' speed="5" text=localized_title_6_11 %}
+        {% include './video-thumb.html' with class="tw-col-span-6 tw-col-start-1 tw-row-start-12" size="small" image='scene6/3.jpg' speed="3" text=localized_title_6_3 %}
```
Comment (tbrlpld) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/pages/youtube-regrets-2022/fragments/timeline.html:199]: See above

**Review by tbrlpld — CHANGES_REQUESTED** [review]
@TheoChevalier I have left some questions, that would be good to clarify.

**Comment by TheoChevalier** [comment]
> @TheoChevalier I have left some questions, that would be good to clarify.

Hey @tbrlpld , there was no specific reason for that, I debated this with myself and didn’t have a strong opinion, I just through grouping them would be more readable overall.
Just pushed an update!

**Review by tbrlpld — APPROVED** [review]

**Comment by tbrlpld** [comment]
> > @TheoChevalier I have left some questions, that would be good to clarify.
> 
> Hey @tbrlpld , there was no specific reason for that, I debated this with myself and didn’t have a strong opinion, I just thought grouping them would be more readable overall. Just pushed an update!

Thanks @TheoChevalier for the clarification. I think this way is a little easier to grasp. Thanks for the update.

**Pull request merged**